### DISCUSSION TITLE
后面会有支持21:9分辨率的版本吗

### DISCUSSION THREAD

**Discussion by LeiWenTong1** [babalae/better-genshin-impact#543] [General]
后面会有支持21:9分辨率的版本吗

**Comment by emako** [comment]
估计有点难，太小众了，重点是维护人员里也没有用这个比例的

**Reply by LeiWenTong1** [reply]
了解，谢谢大佬

**Comment by huiyadanli** [comment]
21:9的关键问题在于。。。mhy为这个分辨率设置了极其特殊的布局。。和原本的响应式规律表现不一致

还有，我没有相关设备进行测试

**Reply by LeiWenTong1** [reply]
谢谢大佬，这个程序我开16:9分辨率试了下，很好用，反应也快，虽然我那个淦饭鼠标宏有很多自动，但大佬的这个用图像识别更智能，不舍得调低分辨率，我以后换了4k屏再用吧
